.github/workflows/create-release.yml

```diff
@@ -0,0 +1,28 @@
+name: Release
+
+on:
+  pull_request:
+    types:
+      - closed
+
+jobs:
+  create-release:
+    name: Create release PR
+    runs-on: ubuntu-latest
+    # Automate releases with Conventional Commit Messages as Pull Requests are merged into "main" branch
+    if: github.event.pull_request.merged == true
+    steps:
+      - name: Run release please action
+        uses: GoogleCloudPlatform/release-please-action@v2
+        with:
+          token: ${{ secrets.GH_TOKEN }}
+          release-type: simple
+          package-name: dsp-api
+          changelog-types: '[
+            {"type": "feat", "section": "Enhancements", "hidden": false },
+            {"type": "fix", "section": "Bug Fixes", "hidden": false },
+            {"type": "build", "section": "Maintenance", "hidden": false },
+            {"type": "chore", "section": "Maintenance", "hidden": false },
+            {"type": "refactor", "section": "Maintenance", "hidden": false },
+            {"type": "docs", "section": "Documentation", "hidden": false }
```

Comment:
What about `test` option we have in template and not here?

Reply:
I don't know to be honest. @subotic  is there a reason why `test` (and also `style`) is missing in this list?

Reply:
`Build` also is obsolete here, I actually added it because someone never used `chore` for the maintenance stuff, and it was missing in the change log later on.

There is no Test section, so we can either or keep it simple and have it under `Maintenance`, the same as refactor.

Reply:
I would say `style`, `test` and `refactor` should go under `Maintenance` (I guess the end user doesn't care about the differences). And I would delete `build` as it is not on "our" list of prefixes (https://docs.dasch.swiss/latest/developers/dsp/contribution/). But as you are dealing with the release notes now, you can decide and I will do whatever you think makes sense.

Reply:
So add these which are missing and let's keep the `build` there as an alias to `chore`, because it's used frequently by Ivan.

Reply:
> I don't know to be honest. @subotic is there a reason why `test` (and also `style`) is missing in this list?

This stems from the idea behind release notes, which are for the end-user. They should only be informed about bug fixes, improvements and new features. The rest is not having any impact for the end-user and thus does not need to be included in the release notes. Since these release notes are not for the end-user but for us, we can do here whatever we like.

Reply:
Also keep in mind that these are not our release-notes. This is a summary of the Github release. Github release does not equal DaSCH DSP release.